### DISCUSSION TITLE
Wrong language guess

### DISCUSSION THREAD

**Discussion by lambdakilo** [stoatchat/discussions#635] [Q&A] [answered]
I'm having issues with https://app.revolt.chat/login . The login screen guesses my language wrong but only just a bit. My language is FI Suomi but the website suggests me PH Filipino. Which repository should I report this bug in?

**Comment by searinminecraft** [comment, marked as answer]
You should report the issue at the [revoltchat/revite](https://github.com/revoltchat/revite) repository. But since the new client is in the works I don't think the issue will be fixed on Revite. But hopefully should be fixed in the new client.
